src/App.tsx

```diff
@@ -1,26 +1,218 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
 
-const USER_ID = 0;
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { TodoTemp } from './components/TodoTemp';
+import { ErrorNotifications } from './components/Errors';
+
+import { Todo } from './types/Todo';
+import { Errors } from './types/ErrorsEnum';
+import { Filter } from './types/FilterEnum';
+import { handleError } from './handlers/handleError';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+  USER_ID,
+} from './api/todos';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(Filter.ALL);
+  const [inputChange, setInputChange] = useState('');
+  const [focusOnChange, setFocusOnChange] = useState(true);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadingTodoIds, setLoadingTodoIds] = useState<number[]>([]);
+  const [errorNotification, setErrorNotification] = useState<Errors>(
+    Errors.DEFAULT,
+  );
+  const [editingTodo, setEditingTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    setErrorNotification(Errors.DEFAULT);
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        handleError(Errors.LOADING, setErrorNotification);
+      });
+  }, []);
```

Comment:
Move the `useEffect `to the end of code
Placing `useEffect `at the end improves readability by letting developers first grasp the main logic before seeing side effects. This promotes clearer structure and consistency in the codebase.